enhancements/cloud-integration/tokenized-auth-enablement-operators-on-cloud.md

```diff
@@ -353,8 +358,34 @@ On AWS STS, the `STSIAMRoleARN` would be something like:
 `arn:aws:iam::269733383066:role/newstscluster-openshift-logging-role-name`<br>
 This provides the role information linked by policy to access various cloud resources.
 
+### Changes for HyperShift
+
+#### CCO Stories
+
+The kubeconfig of the hosted cluster is mounted at a specied path on the CCO 
+pod. CCO can consume that kubeconfig when it starts the go client.
+
+The Pod Identity Webhook is not shipped alongside CCO in order to reduce the 
+footprint of the CCO install.
+
+#### HyperShift Stories
+
+CCO without the pod identity webhook is enabled on the HyperShift management 
```

Comment:
https://github.com/openshift/cloud-credential-operator/pull/547 disables starting the pod identity webhook controller when the infrastructure ControlPlaneTopology is External.